capability/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Overview
+nav_order: 1
+parent: Capability
+grand_parent: RRAP M&DS
+---
+
+{: .no_toc }
+
+# RRAP M&DS Capability
+
+<details  open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+{: .text-delta }
+* TOC
+{:toc}
+____
+</details>
+
+## Our Mission
+
+Our Mission is to help environmental decision makers on the Reef explore and identify robust intervention options under uncertainty. We do this using integrated state-of-the-art environmental, ecological and economic models and decision-support processes.
+
+Our modelling and decision-support (MDS) suite is a unique synthesis of ecological models with information systems and decision-support tools. The MDS suite is designed to help decision makers explore benefits, costs and risks of new and existing interventions under uncertainty.​
+
+We use the best scientific knowledge of global and regional environmental forcing in space and time, and the biological and ecological responses of corals, to inform actions that can sustain a resilient Reef in the face of climate change. ​
+
+We translate ecological information to value functions to identify intervention options that represent viable and lasting economic and social outcomes for Reef and peoples.  ​
+
+## Our Goal
+
+Our Goal is to develop an operational suite of models and decision-support systems that:
+
+-   Produce forecasts of reef conditions and values for the GBR under different climate change scenarios & traditional management systems​ ​
+-   Enable insights into intervention solutions that can deliver robust outcomes against multiple objectives in space and time under uncertainty, constraints and limited resources​
+-   Support informed strategic and tactical intervention decision-making and management planning for Reef, people and industries
+
+## RRAP models and decision-support systems (MDS) inform strategic and tactical Reef interventions
+
+The MDS suite is a collaborative  set of environmental, ecological, and economic models working within a decision-support framework. Climate models force the environmental and ecological layers, and influence intervention scope and effectiveness. Uncertainty is propagated throughout the information supply chain.  Scenario analyses of ecological and economic outcomes feed back to inform which intervention strategies and tactics are effective, safe and robust.  ​
+
+
+[**Reef Characterisation and Selection**](./reef-charact-select/index.md){:target="\_blank"}: Capabilities to assist with reef characterisation and selection. 
+
+[**Counterfactuals**](./counterfactuals/index.md){:target="\_blank"}: Modelling of potential futures for reefs under various uncertainties.
+
+[**Intervention Modelling**](./intervention-modelling/index.md){:target="\_blank"}: Characterisation of the effect of interventions on reefs.
```

Comment:
The links should either end with ".html" or just not include a file extension (I'd recommend no file extension).

Reply:
Ok, so I'll just remove "index.md" from them all.
So link should then default to the "index" file I'm guessing?